guardowl/protocols.py

```diff
@@ -874,3 +755,177 @@ def run_DOF_scan(
         raise NotImplementedError
     elif DOF_definition["torsion"]:
         raise NotImplementedError
+
+
+def run_detect_minimum_test(
+    nnp: str,
+    implementation: str,
+    platform: Platform,
+    output_folder: str,
+    percentage: int = 10,
+    only_molecules_below_10_heavy_atoms: bool = False,
+) -> Dict[str, Tuple[float, float]]:
+    """
+    Perform a minimization on a selected compound.
+    :param nnp: The neural network potential to use.
+    :param implementation: The implementation to use.
+    :param name: The name of the molecule to simulation.
+    """
+    from guardowl.testsystems import SmallMoleculeTestsystemFactory
+    import mdtraj as md
+    from .utils import (
+        extract_drugbank_tar_gz,
+        _generate_file_list_for_minimization_test,
+        _generate_input_for_minimization_test,
+    )
+
+    # extract drugbank tar.gz file
+    extract_drugbank_tar_gz()
```

Comment:
Is there a benefit to redistributing the drugbank tar file? There is a package called drugbank-downloader which can get drugbank data without redistributing it. I am not in favour of either method but just wanted to offer an alternative that does not add the tar file to the repository as some people like to have a light and streamlined repository.

Reply:
that is a good point. It might be better to have a separate repository with the durgbank tar archive and downloading it when the test is required. The reason why I prefer the tar archive is because we also require the optimized files. I will open an issue about that.